src/misc/documentation_styleguide.md

```diff
@@ -85,13 +124,19 @@ When adding images to a repository, make sure that the size is not too big for t
 
 ## Tools
 
-[Atom](https://atom.io/) is the best tool for editing Markdown files:
-* You can preview the content with `CTRL+SHIFT+M`. If you forget the shortcut, you can always use `CMD+SHIFT+P` to display the list of available commands, and search for `Markdown Preview: Toggle`.
-* You can install a package called [smart-quotes-plus](https://atom.io/packages/smart-quotes-plus), which lets you swap straight quotes with curly ones. Once the package is installed, press `CTRL+ALT+'` to swap quotes on the entire document, or the text currently selected. Again, you can use `CMD+SHIFT+P` and search for `Smart Quote Plus: Smartreplace`. Only make sure to not swap `"` or `'` in code fragments where these characters are wanted.
+[Visual Studio Code](https://code.visualstudio.com/) is the best tool for editing Markdown files:
+* You can preview the content with `⇧⌘V` or `Ctrl+Shift+V` (or right-click on the tab and select `Open Preview`). You can also use `⇧⌘P` or `Ctrl+Shift+P` to display the list of available commands, and search for `Markdown: Open Preview`.
+* You can install a package called [Smart Quotes](https://marketplace.visualstudio.com/items?itemName=LukeParkinson.smart-quotes), which lets you swap straight quotes with curly ones. Once the package is installed, use `⇧⌘P` or `Ctrl+Shift+P` and search for `Convert smart quotes`. Make sure to not swap `"` or `’` in code fragments where these characters are wanted.
+
+VS Code allows the user to [set different personal settings](https://code.visualstudio.com/docs/getstarted/settings): *User settings* (global personal settings) and *Workplace setting* (settings specific for a project that override user settings). These are some useful settings for working with Markdown files:
+* Enable `Editor: Trim Auto Whitespace` to trim automatically trailing whitespaces.
+* Enable `Files: Insert Final Newline` and `Files: Trim Final Newlines` to automatically have leave only one line at the end of files.
```

Comment:
Nit, extra "have" here: "(...) to automatically have leave only one line at the end of files."